psdk_interfaces/srv/CameraDownloadFileList.srv

```diff
@@ -4,4 +4,5 @@ uint8 payload_index 1   # see enum E_DjiMountPosition in dji_typedef.h
 ---
 #response
 bool success
-int16 result            # total_count from enum T_DjiCameraManagerFileList in dji_camera_manager.h
\ No newline at end of file
+ImagesInfo result       # vector of image info
+int16 count            # total_count from enum T_DjiCameraManagerFileList in dji_camera_manager.h
```

Comment:
it is more readable if you move the comments on top of the line that you are describing or commenting on